grant/jwtbearer/handler.go

```diff
@@ -0,0 +1,232 @@
+package jwtbearer
```

Comment:
If i understand correctly, handlers in Hydra is tested with cypress tests. I am willing to write one, but i have some problems with it, as i stated before:  i don't know how to generate jwk from cypress and it is needed to create grant. Maybe you have some ideas? Maybe we can generate it beforehand and store it somewhere inside repository?

Reply:
They sort of are, but not really. e2e tests are just there to validate no problems with cookies and flows and so on. The tests for the handlers them selves usually cover many more use cases and failure cases because they execute faster.

So here, I would expect tests similar to: https://github.com/ory/hydra/blob/master/client/sdk_test.go

Reply:
Regarding key generation in javascript, you might be able to use a library like (https://www.npmjs.com/package/node-jwk). It doesn't look like they are really good but hey for testing it's probably enough. Checking in a predefined key is also fine from my side: https://mkjwk.org

Alternatively you could make a request to hydra's JWK url to generate a key.